File: examples/uint8_array.js
const array = new Uint8Array(10)
console.log(array)

const offset = 1

array.set([1, 2, 3], offset)
console.log(array)

const sub = array.subarray(2)
console.log(sub)

const sub2 = array.subarray(2,4)
console.log(sub2)

console.log(array.byteLength)

File: examples/uint8_array.go
package main

import "fmt"

func main() {
	array := make([]byte, 10)
	fmt.Println(array)

	offset := 1

	copy(array[offset:], []byte{1, 2, 3})
	fmt.Println(array)

	sub := array[2:]
	fmt.Println(sub)

	sub2 := array[2:4]
	fmt.Println(sub2)

	fmt.Println(len(array))
}
